Duplicati/Server/Program.cs

```diff
@@ -302,10 +303,54 @@ public static int RealMain(string[] _args)
             }
 
             if (UpdatePoller != null && UpdatePoller.IsUpdateRequested)
+            {
+                if (Platform.IsClientWindows && !String.IsNullOrWhiteSpace(WindowsService.ServiceControl.SERVICE_NAME) && WindowsServiceExists(WindowsService.ServiceControl.SERVICE_NAME))

```

Comment:
This checks only if the Duplicati service _is installed_. The proposed check:

1. does not check if the Duplicati service is running;
2. does not check that the process currently executing is in fact the service.

I had a look at `Duplicati\WindowsService\ServiceControl.cs`. It seems that `ServiceControl` creates a new `Duplicati.Service.Runner' instance, which in turn starts a new `Duplicati.Server.exe` process. Perhaps the `Duplicati.Server.exe` process should check:

1. am I a child process of `Duplicati.WindowsService.exe`?
2. is the parent process running as a service and what is its Process Id?